### ISSUE TITLE
Nested routes are not supported

### ISSUE THREAD

**Issue by maoberlehner** [wobsoriano/numix#5]
Problem:

A huge part of what makes Remix great is loading of nested data. Unfortunately, as of now, this seems not to be supported by this plugin yet.

Reproduction:

- Create a `pages/foo.vue` file
- Create a `pages/foo/[slug].vue` file
- Add loaders to both
- Navigate to `foo/1`
- Only the loader of the root page component is resolved

See: https://stackblitz.com/edit/nuxt-starter-mwn1xp?file=pages%2Ffoo%2F[slug].vue&initialpath=/foo/1

**Comment by wobsoriano** [comment]
Nested routes supported have been added! 

Sorry for the late response 😆 

https://stackblitz.com/edit/nuxt-starter-euxwj6?file=pages%2Ffoo.vue

**Comment by maoberlehner** [comment]
No worries ;)

Unfortunately it doesn't work with the Stackblitz you provided: https://stackblitz.com/edit/nuxt-starter-euxwj6?file=pages%2Ffoo.vue&initialpath=/foo/1 (`Root:` should be `Root: bar`)

**Comment by aaronksaunders** [comment]
any expected date for this issue to be resolved? This is a pretty interesting project, thanks in advance

**Comment by wobsoriano** [comment]
Hey @aaronksaunders, still trying to figure out how to get the exact route name of a nested route.

Currently using `useRoute().name` to generate a unique key to pass to `useAsyncData`. You can see it [here](https://github.com/wobsoriano/numix/blob/48595211a54b101f2f15174f78d3eea4d6b6ff86/src/runtime/composables/other.ts#L15). This doesn't work on nested routes, as it returns the current route name to all components accessing it.

Have you checked [form-actions-nuxt](https://github.com/Hebilicious/form-actions-nuxt)?

**Comment by aaronksaunders** [comment]
Yes, I am trying to work through some issues there also, got the nested
routes working I think, now trying to get form actions going.
